Makefile

```diff
@@ -178,6 +183,9 @@ const defaultRuntimeRun = "$(PKGRUNDIR)"
 const defaultShimPath = "$(SHIMPATH)"
 const pauseBinRelativePath = "$(PAUSEBINRELPATH)"
 
+const defaultDefaultVCPUs uint32 = $(DEFDEFVCPUS)
```

Comment:
There is already a function called defaultVCPUs(). Wouldn't it be confusing having both with the same name? If not, then I can change as you propose.